packages/liquidator/src/liquidator.js

```diff
@@ -6,25 +6,47 @@ const {
   revertWrapper
 } = require("@umaprotocol/common");
 
+const { ETH_ADDRESS } = require("./constants");
+
 class Liquidator {
   /**
    * @notice Constructs new Liquidator bot.
    * @param {Object} logger Module used to send logs.
+   * @param {Object} oneInch Module used to swap tokens on oneInch.
    * @param {Object} expiringMultiPartyClient Module used to query EMP information on-chain.
    * @param {Object} gasEstimator Module used to estimate optimal gas price with which to send txns.
    * @param {Object} votingContract DVM to query price requests.
+   * @param {Object} syntheticToken Synthetic token (tokenCurrency).
    * @param {Object} priceFeed Module used to query the current token price.
    * @param {String} account Ethereum account from which to send txns.
+   * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
    * @param {Object} empProps Contains EMP contract state data. Expected:
    *      { crRatio: 1.5e18,
             minSponsorSize: 10e18,
             priceIdentifier: hex("ETH/BTC") }
    * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
    */
-  constructor(logger, expiringMultiPartyClient, votingContract, gasEstimator, priceFeed, account, empProps, config) {
+  constructor(
+    logger,
+    oneInch,
+    expiringMultiPartyClient,
+    gasEstimator,
+    votingContract,
+    syntheticToken,
+    priceFeed,
+    account,
+    empProps,
+    config
+  ) {
     this.logger = logger;
     this.account = account;
 
+    // Initial version will only allow ETH to be the reserve
+    this.reserveCurrencyAddress = ETH_ADDRESS;
+
+    // OneInch module
+    this.oneInch = oneInch;
```

Comment:
It's not clear to me from reading the input vars whether `oneInch` should be a contract or a client (like EMPClient), so I think you should consider renaming this to `oneInchClient`